examples/sections/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="css/ul.css">
     <link rel="stylesheet" href="css/app.css">
 
-    <script src="bower_components/dom-scheduler/lib/dom-scheduler.js" defer></script>
+    <!-- <script src="bower_components/dom-scheduler/lib/dom-scheduler.js" defer></script> -->
```

Comment:
@etiennesegonzac do we want the example with or without dom-scheduler in the document?